tempto-core/src/main/java/com/teradata/tempto/internal/fulfillment/ldap/LdapObjectEntryManager.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.teradata.tempto.internal.fulfillment.ldap;
+
+import com.teradata.tempto.fulfillment.ldap.LdapObjectDefinition;
+
+import java.util.List;
+
+interface LdapObjectEntryManager
+{
+    List<String> addLdapDefinitions(List<LdapObjectDefinition> ldapObjectDefinitions);
```

Comment:
If an entry already exist, we just ignore it for now. We want to actually get some tests running with this, and if its feel necessary I can add it in a separate patch.